### PR TITLE
Update release notes and meta data for 0.22.4

### PR DESCRIPTION
**Pull request by HeroicKatora**
While the error changes for `0.23` are still a work in progress (soon!) the master branch has accumulated enough bug fixes and other improvements to warrant another minor release.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,17 @@ Rust image aims to be a pure-Rust implementation of various popular image format
...
+### Version 0.22.4
+
+- Added in-place variants for flip and rotate operations.
+- The bmp encoder now checks if dimensions are valid for the format. It would
+  previously write a subset or panic.
+- Removed deprecated implementations of `Error::description`
+- Added `DynamicImage::into_*` which convert without an additional allocation.
+- The PNG encoder errors on supported color types where it had previously
+  silently mixed color channels.
+- Enabled saving images as `gif` and `dxt` with `save_buffer`.
```
Comment (fintelia) [inline CHANGES.md]: Didn't we drop dxt save support before merging the PR?

File: CHANGES.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ Rust image aims to be a pure-Rust implementation of various popular image format
...
 ## Changes
 
+### Version 0.22.4
+
+- Added in-place variants for flip and rotate operations.
+- The bmp encoder now checks if dimensions are valid for the format. It would
+  previously write a subset or panic.
+- Removed deprecated implementations of `Error::description`
+- Added `DynamicImage::into_*` which convert without an additional allocation.
+- The PNG encoder errors on supported color types where it had previously
```
Comment (fintelia) [inline CHANGES.md]: supported -> unsupported ?

Comment (HeroicKatora) [reply CHANGES.md]: Yes. I've also changed the wording to `swapped color channels` to avoid the confusion of alluding to actual color mixing.

**Comment by HeroicKatora** [comment]
The version has been [published](https://crates.io/crates/image/0.22.4).

**Comment by HeroicKatora** [comment]
Note: if you look at the [git-tag](https://github.com/image-rs/image/releases/tag/v0.22.4) you will also find that I tried out a new thing. In theory nothing binds the tar-ball available at crates.io to the source repository other than a simple (usually autogenerated) text file that specifies the purported git revision from which it was packaged. Now, I've done the reverse and *signed* within the repository the package that was created locally on my machine. Maybe I'll setup automated tooling for this.

**Pull request merged**